components/infieldprogresscircle/index.css

```diff
@@ -0,0 +1,36 @@
+/*!
+Copyright 2025 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+.spectrum-InfieldProgressCircle {
+	--mod-progress-circle-thickness: var(--mod-progress-cirlce-stroke-width, 2px);
```

Comment:
This 2px- I see in Figma that there's a token `progress-circle-thickness-small`. Should we be using that instead of the hardcoded 2px? 

Also, do we want to introduce a mod for this? Are we creating a mod for a mod? 🙃 I don't know the answer to that question, but is this infield progress circle supposed to just overwrite any of the progress circle mods, including `--mod-progress-circle-stroke-width`? Should this line just be `--mod-progress-circle-thickness: var(--spectrum-progress-circle-thickness-small)`?

And if not- I caught another misspelling of circle in that `--mod-progress-circle-stroke-width`. Those pesky L's and C's 😆